site/en/blog/immutable-document-domain/index.md

```diff
@@ -0,0 +1,197 @@
+---
+layout: 'layouts/blog-post.njk'
+title: "`document.domain` is becoming immutable"
+description: >
+  Websites will be unable to set `document.domain` on Chrome starting in version 101. If your website relies on setting `document.domain`, your action is required.
+subhead: >
+  Beginning with Chrome version 101, websites will be unable to set `document.domain`.
+date: 2022-01-06
+authors:
+  - agektmr
+tags:
+  - security
+hero: 'image/YLflGBAPWecgtKJLqCJHSzHqe2J2/grGMzuhOjsGhN150dONe.jpg'
+alt: >
+  A dog in disguise.
+---
+
+**If your website relies on relaxing the same-origin policy via `document.domain`,
```

Comment:
@heyawhite Switch formatting to an appropriate banner component https://web.dev/handbook/web-dev-components/#banners

Reply:
See https://github.com/GoogleChrome/developer.chrome.com/pull/1896#discussion_r778761636